packages/cli/src/cmds/agentInstaller/install-agent.ts

```diff
@@ -231,6 +231,8 @@ export default {
     }
 
     if (errors.length) {
+      // Make sure there's nothing left in the queue before we exit.
+      await Telemetry.flush();
       Yargs.exit(1, new Error(errors.map((e) => e.message).join('\n')));
     }
```

Comment:
```suggestion
    // Make sure there's nothing left in the queue before we exit.
    await Telemetry.flush();
    if (errors.length) {
      Yargs.exit(1, new Error(errors.map((e) => e.message).join('\n')));
    }
```
We send telemetry events when things succeed, too. Shouldn't this flush happen unconditionally?

Reply:
Whoops. Yes. I changed this last minute. Thanks for catching that.